### PR TITLE
[DOC-3280] Add SMP what's supported topic

### PR DESCRIPTION
**Pull request by brian-f-harness**
Add SMP what's supported topic

For reviewers, [a preview](https://648b4764969c0c296f0dc535--harness-developer.netlify.app/docs/self-managed-enterprise-edition/smp-supported-platforms) is available.

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by pratmit — APPROVED** [review]
LGTM

File: docs/self-managed-enterprise-edition/smp-supported-platforms.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+description: Platforms and technologies supported by Self-Managed Enterprise Edition
+sidebar_label: What's supported
+sidebar_position: 1
+---
+
+import Smp from '/docs/self-managed-enterprise-edition/shared/smp-supported-platforms.md';
+
+<Smp />
+
+For information about what's supported for features and integrations, go to [Supported features and integrations](/docs/self-managed-enterprise-edition/introduction/supported-features-and-integrations.md).
```
Comment (aimurphy) [inline docs/self-managed-enterprise-edition/smp-supported-platforms.md:12]: Not sure if you meant to do this, but this link is duplicated. You have this sentence near the top of your partial as well as here.

File: docs/self-managed-enterprise-edition/smp-supported-platforms.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+title: What's supported in Self-Managed Enterprise Edition
+description: Platforms and technologies supported by Self-Managed Enterprise Edition
+sidebar_label: What's supported
+sidebar_position: 1
+---
+
+import Smp from '/docs/self-managed-enterprise-edition/shared/smp-supported-platforms.md';
+
+<Smp />
+
```
Comment (aimurphy) [inline docs/self-managed-enterprise-edition/smp-supported-platforms.md:11]: You might want to add a heading like `### See also` or something, so these paragraphs are distinct from the `Terms of support` section.

**Review by aimurphy — APPROVED** [review]
Approved w some comments for consideration.

**Pull request merged**